uvicorn/_types.py

```diff
@@ -29,3 +29,39 @@ class LifespanSendMessage(TypedDict):
         "lifespan.shutdown.failed",
     ]
     message: Optional[str]
+
+
+class HTTPScope(TypedDict):
```

Comment:
cant we import the types from asgiref directly ?

Reply:
I would have to pip install from the latest asgiref branch, because that feature was not released yet. Do you prefer?